prow/auth-presubmit.sh

```diff
@@ -40,7 +40,6 @@ else
 fi
 
 echo "=== Bazel Build ==="
-./bin/install-prereqs.sh
```

Comment:
Is it always true that we don't need it?

Reply:
this just install gometalinter which is not needed.